mods/ts/rules/misc.yaml

```diff
@@ -94,6 +96,8 @@ CRATE:
 		SelectionShares: 5
 		Condition: crate-speed
 		Notification: 00-i080.aud
+	EditorTilesetFilter:
+		Categories: Crate
```

Comment:
Crates have a limited lifetime so can't really be placed on the map.  This makes them a good candidate for the "System" category I suggested above.